Core/scripts/dirac-create-distribution-tarball.py

```diff
@@ -463,10 +458,9 @@ def __compileReleaseNotes( self, rstFile ):
     for baseFileName in baseList:
       htmlFileName = baseFileName + ".html"
       try:
-        fd = open( htmlFileName, "w" )
-        fd.write( parts[ 'whole' ] )
-        fd.close()
-      except Exception, excp:
+        with open( htmlFileName, "w" ) as fd:
+          fd.write( parts[ 'whole' ] )
+      except Exception as excp:
         return S_ERROR( "Could not write %s: %s" % ( htmlFileName, excp ) )
```

Comment:
correct, not very nice to have the "," there but ;-)
repr(x).replace(',)',')')   ;-))